src/client/cytoscape-extensions.js

```diff
@@ -11,7 +11,7 @@ const compoundCollapse = require('cytoscape-compound-collapse');
 const fisheye = require('cytoscape-fisheye');
 
 //Tooltips
-// const tippy = require('cytoscape-tippy');
+const popper = require('cytoscape-popper');
```

Comment:
@hmtinc Do you know if the build file for cytoscape-popper is committed in the origin repository?

Reply:
https://github.com/cytoscape/cytoscape.js-popper/commit/b3cdcab91014b63ae2a86d794d1ac7e688a54c36
I think the build file needs to be excluded from the gitignore